content/blog/2021/announcing-pycon-india.md

```diff
@@ -0,0 +1,43 @@
+Title: PyCon India is back online for 2021
+Date: 2021-02-25 17:00
+Author: Nikhil Maan (@Sc0rpi0n101)
+Category: 2021
+Slug: announce-pycon-india-2021
+Summary: Announcing the return of PyCon India as an online conference for 2021.
+
+We hope you've had a safe and healthy year. 
+
+We are very excited to announce that [**PyCon India**](https://inpycon21.netlify.app/) is back online for 2021.
+
+Get ready for 4 days of non-stop fun and learning with 4 conference tracks, sessions, BOFs, community booths, workshops, and devsprints.
+
+Accessing the situation with COVID-19 and the rate of vaccination, we have decided to keep PyCon India 2021 as an online event to ensure the safety of everyone involved. We know you all are excited to meet everyone in person and celebrate Python in the "real world", but everyone's health and safety are the utmost priority at this time. 
+
+## Dates
+
+The conference will be held from September 16 to 19, 2021.
+
+* Conference Days: September 16 and 17
+* Workshops: September 18
+* Devsprints: September 18 and 19
+
+**Ticket sales will be going live on April 15, 2021.**
+
+** The format for devsprints is still being planned and can be changed anytime before the conference. 
+
+## Platform
+
+With our amazing experience last year, and great feedback from the attendees, we have decided to use [Hopin](https://hopin.com/) as the platform for the conference again. 
+
+## Call for Proposals
+
+We will be starting our call for proposals soon.  In case you are excited about teaching or talking about Python, no matter whether you started recently or have been using Python for a decade, make sure you put in a talk proposal. No one became an expert in a day - neither at presenting nor at programming. So don't be shy and don't think too much, just put in a proposal.
+
+* CFP Opens: March 01
+* CFP Closes: May 15
+
```

Comment:
```suggestion
* CFP Opens: 1st March 2021
* CFP Closes: 15th May 2021

```

Do we really want these here after what happened last year? Changing or extending dates gets people confused and then reach out that they thought X was the data for Y

Reply:
It was public on the inpycon2021-tasks readme, so I added it. We can remove it if you want and release the dates with the CFP announcement.

Reply:
```suggestion
* CFP Opens: First week of March 2021
* CFP Closes: 15th May 2021

```